telegram/ext/dispatcher.py

```diff
@@ -193,6 +192,7 @@ def start(self):
             self.logger.error(msg)
             raise TelegramError(msg)
 
+        self._init_async_threads(uuid4(), self.workers)
```

Comment:
I think I agree with this change, but I'd like @tsnoam to give it a look as well.